CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
+
+* Bug fixes:
+  * Fix an Azure NPM issue that user could meet unexpected connectivity for Pods on the Node when editing a NetworkPolicy with a CIDR "except" field.
+  * Fix a bug where users could not stop/start AgentPools using the PowerState field in the 2020-09-01, 2020-11-01, 2020-12-01, 2021-02-01, 2021-03-01 API versions.
+  * Fix bug to block non-VMSS (VirtualMachineScaleSets) agent pools in the Automatic SKU validation process.
+  * Fix bug to ensure correct default network plugin settings for Kubernetes clusters using VMAS.
+  * Fix bug for intermittent precondition failures when applying an AKS Bicep deployment on the pod subnet delegation.
+  * Fix bug of public IP on VMSS dropped after upgrade node image or reset service principal operation.
+  * Fix bug (#4282)[https://github.com/Azure/AKS/issues/4282] to remove duplicated toleration from Calico components.
+  * Fix bug to ensure AnnotationControlled is correctly populated by default when creating AKS clusters with app routing enabled, and to ensureAnnotationControlled is an accepted value for the default nginx ingress controller config for AKS clusters with K8s versions <1.30.
+  * Fix bug for [karpenter](https://github.com/Azure/karpenter-poc/issues/639).
+
+* Behavior change:
+  * Cluster Stop operations will be blocked if the customer has any illegal webhooks:
+    * If APIGroups is wildcard, and Resources is in the criticalResources or wildcard, the webhook will be blocked.
+    * If Resources is wildcard, and APIGroups is in the criticalApiGroups or wildcard, the webhook will be blocked.
+  * For non-host network pods running on AKS nodes, they cannot access wireserver(168.63.129.16) port 32526. Before this change user cannot access wireserver port 80, but port 32526 is accessible.
+  * When deploying an [AKS Automatic (preview)](https://learn.microsoft.com/en-us/azure/aks/intro-aks-automatic) cluster, user do not need to register extra feature flags for related preview features, such as APIServerVnetIntegration, NRGLockdown, NodeAutoProvisioning, and Safeguards.
+  * Allow creation of AgentPools without Count field specified if autoscaler enabled.
+  * CBL-Mariner 1.0 is end of life, creation of new nodepools with OSSKU cblmariner is disabled.
+  * Azure Linux 2.0 on Kubernetes version 1.30 LTS will be blocked until Azure Linux 3.0 is released and supported.
+  * KMS v1 is blocked for AKS cluster with K8s version >=1.31.
+  * [AGIC addon](https://learn.microsoft.com/en-us/azure/application-gateway/ingress-controller-overview) has been assigned the network contributor role.
+
+* Component updates:
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-202408.27.0](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSUbuntu/gen1/2204containerd/202408.27.0.txt).
+  * Azure Linux image has been updated to [AzureLinux-202408.27.0](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSCBLMarinerV2/gen2/202408.27.0.txt). 
```

Comment:
we need to copy these vhd notes into vhd-notes folder under the AKS repo (last release notes for reference) and then do relative reference